src/tetris/biasEngine/biasEvent.ts

```diff
@@ -0,0 +1,40 @@
+/// <reference path="../../../definitions/phaser.d.ts"/>
+
+import BiasEventType from "tetris/biasEngine/biasEventType"
+
+export default class BiasEvent {
+
+	//region public members
+	public get eventType(): BiasEventType {
+		return this._eventType;
+	}
+
+	public get startTime(): number {
```

Comment:
Why start time? From my point of view the only necessary information is when a bias should stop influencing.
It shouldn't be the responsibility of any other component except the bias engine to schedule bias events.
@Johannes-Schneider We discussed this already. Any changes in plans I missed?

Reply:
I just thought of it as some kind of internal documentation. So maybe we would like to build some kind of timeline later one... But we can also remove it for now.